kybra/compiler/kybra_generate/src/generators/kybra_serde.rs

```diff
@@ -24,7 +24,7 @@
 
```

Comment:
Delete this whole thing

Reply:
Done in 55d1d1dba73234b8b83a0c26c7166f437fe6f7de